README.md

```diff
@@ -32,7 +32,27 @@ contribuição, mas sinta-se à vontade para contribuir:
 Instalação
 ----------
 
-_TODO_ (#12).
+Por ora, esse processo de instalação se orienta a desenvolvedores, 
+em breve criaremos ferramentas mais apropriadas de acesso à informação para o público geral.
+
+Primeiras coisas primeiro:
+- `sudo apt update`
+- `sudo apt install git curl`
```

Comment:
Desnecessário neste caso.

Por sinal, se for colocar comandos, por favor não coloque comandos específicos de um gerenciador de pacote. Se for fazê-lo, então especifique: "".

```suggestion
Primeiras coisas primeiro, garanta que sua distribuição está atualizada. Para distribuições baseadas em Debian (ex: Ubuntu, Mint), execute como _root_:

```console
# apt update
# apt install git curl
```
```